demisto_sdk/commands/content_graph/objects/base_playbook.py

```diff
@@ -48,6 +204,15 @@ def prepare_for_upload(
     def _client_upload_method(cls, client: demisto_client) -> Callable:
         return client.import_playbook
 
+    def save(self):
+        super().save()
```

Comment:
shouldnt call save after?

Reply:
Nope, we want to override the dict once we're done.